gui/default/index.html

```diff
@@ -248,6 +248,32 @@ <h3 class="panel-title">
       </div>
     </div>
 
+    <!-- Panel: Unacked Notifications -->
+
+    <div ng-if="config && config.options && config.options.unackedNotifications" ng-repeat="message in config.options.unackedNotifications track by $index" class="row">
+      <div class="col-md-12">
+        <div class="panel panel-warning">
+          <div class="panel-heading">
+            <h3 class="panel-title">
+              <div class="panel-icon">
+                <span class="fa fa-exclamation-circle"></span>
+              </div>
+              <span translate>Notification</span>
+            </h3>
+          </div>
+          <div class="panel-body">
+            <p>{{ message }}</p>
```

Comment:
Yep, alternative have some magic ids or something?
Also unlikely the messages will get translated as fast as we produce them.